float64 timeStamp
float64 leftEncoder
float64 rightEncoder